---
layout: post
title:  "Comparison of DNA Extraction Methods"
date:   2020-05-12
categories: white_paper
permalink: /white_paper/Comparison-of-DNA-Extraction-Methods
toclink: /OCMS-blog/white_paper/Comparison-of-DNA-Extraction-Methods
author: Sam Bullers
contributors: Sam Bullers, Rachel Feeney, Matt Jackson
excerpt_separator: <!--more-->
updated: today
color: blue
parent: White Papers
parent_url: /OCMS-blog/white_paper
postid: white_001
---

## Executive Summary
The pilot study reported here aimed to determine which commercially available DNA extraction kit and homogenisation step performed the best when extracting DNA from the ZymoBIOMICS Community Standard which is comprised of 10 microbes in known quantities.

As the ZymoBIOMICS standard is a known defined community, the best extraction kit and homogenisation method will be the steps which results in the sequencing data closest to the initial input.

The Community Standard was diluted from neat to 1:1000000 to determine if a particular extraction kit or homogenisation step yielded greater DNA concentrations and at what point DNA was no longer detected from each variable. DNA quantification was performed on a Nanodrop and all kits showed recovery of good quantity and quality DNA from both kits and both homogenisation steps. For all conditions, DNA concentration hit a base level at a 1:100 dilution.

Shotgun sequencing was performed on one replicate of the undiluted and 1:10 dilution of each extraction method and homogenisation method. From this data, we observed all conditions tested recovered all of the microbes which make up the community standard. PCA analysis showed that sample concentration (undiluted vs 1:10) or extraction kit (Qiagen vs ZymoBIOMICS) showed no significant difference in microbial DNA recovered. A significant difference in the microbial DNA composition was observed between homogenisation protocols (Tissue Lyser vs Vortex Adapter).

<!--more-->
## Objective
To establish which commercially available DNA extraction kit and homogenisation protocol will be used by the Oxford Centre for Microbiome Studies for all future routine DNA extractions from faecal samples.

## Aims
* Determine optimum DNA extraction kit between the Qiagen QIAmp PowerFecal Pro (#51804) and the ZymoBIOMICS DNA Microprep kit (#D4301) with regard to DNA yield, community standard DNA abundance recovery and general usability.  
* Determine optimum homogenisation protocol between a higher-powered homogeniser (Qiagen Tissue Lyser II) and lower-powered homogeniser (Qiagen Vortex adapter) with regard to DNA yield, community standard DNA abundance recovery and usability.
* Understand at which dilution of the serially diluted community standard, the Nanodrop no longer detects DNA and if diluting the standards affects which species within the standard are detected.  

## Methodology
### Microbial Community Standard
ZymoBIOMICS Microbial Community Standard (#D6300) was used as a defined microbial input into this series of experiments. For an extraction protocol to be successful, the sequencing output will give the same species of bacteria, in the same abundance, as the input. Any bias introduced by the extraction protocol, homogenisation protocol or dilution of the input will be detected by a bias in the sequencing output.

The theoretical composition of the ZymoBIOMICS Microbial Community Standard based on Genomic DNA: _Listeria monocytogenes_ - 12%, _Pseudomonas aeruginosa_ - 12%, _Bacillus subtilis_ - 12%, _Escherichia coli_ - 12%, _Salmonella enterica_ - 12%, _Lactobacillus fermentum_ - 12%, _Enterococcus faecalis_ - 12%, _Staphylococcus aureus_ - 12%, _Saccharomyces cerevisiae_ - 2%, and _Cryptococcus neoformans_ - 2%. Also see [Figure 1](#zymo_std).

<a name="zymo_std"></a>
![zymo_std]({{ site.baseurl }}/assets/images/white_001/zymoBIOMICS_std.png)
**Figure 1. [zymoresearch.com][zymo] graphical illustration of the composition
 of the ZymoBIOMICS Microbial Community Standard.**

### Experimental workflow
The ZymoBIOMICS Microbial Community Standard was diluted 1:10 from neat to 1:1000000 in phosphate buffered saline (PBS). Diluted samples were then split into samples which would be extracted using the Qiagen QIAmp PowerFecal Pro (#51804) or the ZymoBIOMICS DNA Microprep kit (#D4301).

![dilution_series]({{ site.baseurl }}/assets/images/white_001/dilution_series.png)

Samples were then further subdivided into those which were homogenised using the Qiagen Vortex Adapter for 20 minutes, as per the manufacturer’s instructions or homogenised using the Qiagen Tissue Lyser II for 2x5 minutes, with a rotation of samples in between, as per the manufacturer’s instructions ([Figure 2](#experiment_design)). All sample conditions were performed in duplicate.

<a name="experiment_design"></a>
![experiment_design]({{ site.baseurl }}/assets/images/white_001/experiment_design.png)
**Figure 2. Sample workflow**

A complete list of samples can be found in table 1. Pilot 4 investigated both extraction kits using the Qiagen Vortex Adapter. Pilot 5 investigated both extraction kits using the Qiagen Tissue Lyser II.

<table>
  {% for row in site.data.INT001_sample_list %}
    {% if forloop.first %}
    <tr>
      {% for pair in row %}
        <th>{{ pair[0] }}</th>
      {% endfor %}
    </tr>
    {% endif %}
    {% tablerow pair in row %}
      {{ pair[1] }}
    {% endtablerow %}
  {% endfor %}
</table>


### Extraction protocols
Half of the samples in this experiment were extracted using Qiagen QIAmp Power Fecal Pro analytical procedure with modifications to manufacturer's instructions at the sample homogenisation step. Within this group, half the samples were homogenised with the Qiagen Vortex Adaptor on the Votex Genie, for 20 minutes. The other half were homogenised using the Tissue Lyser (5 min, turn, 5 min, at 25 Hz). After sample homogenisation, all samples were processed according to Qiagen's instructions.

The other half of samples in this experiment were extracted using ZymoBIOMICS DNA Microprep kit analytical procedure with modifications to manufacturer's instructions at the sample homogenisation step. Within this group, half the samples were homogenised with the Qiagen Vortex Adaptor, and the other half with the Tissue Lyser, as in the same manner as described above. After sample homogenisation, all samples were processed according to Zymo's instructins.

All DNA concentrations were measure by Nanodrop before storing.

## Results

### Qiagen Vortex Adapter
Homogenisation with the Qiagen Vortex Adapter resulted in good DNA yields from the undiluted ZymoBIOMICS Community Standard. Initial differences between kits in the undiluted standard DNA concentration was likely due the Qiagen PowerFecal kit eluting in 50uL elution buffer while the ZymoBIOMICS extraction kit recommends eluting in 20uL elution buffer. Qiagen PowerFecal kit showed very little variability between replicates (n=2) compared to the ZymoBIOMICS extraction kit for the undiluted sample.  

From a 1:10 dilution both kits produced a similar DNA concentration. From 1:100 dilution onward, both kits had reached a baseline just above the limit of detection for all dilutions ([Figure 3](#vortex_homogenise)).

<a name="vortex_homogenise"></a>
![vortex_homogenise]({{ site.baseurl }}/assets/images/white_001/vortex_adapter_homogenisation.png)
**Figure 3. Nanodrop DNA quantification of DNA extracted from a serial dilution of the ZymoBIOMICS Microbial Community Standard using the Qiagen Vortex Adapter**

### Qiagen Tissue Lyser II
Homogenisation with the Qiagen Vortex Adapter resulted in good DNA yields from the undiluted ZymoBIOMICS Community Standard. The ZymoBIOMICS extraction kit produced comparable DNA concentrations on the Tissue Lyser II to that on the Qiagen Vortex Adapter. However, the Qiagen extraction kit showed lower DNA concentration values from the undiluted standard compared to the Qiagen Vortex Adapter. Again, differences between kits in the undiluted standard DNA concentration was likely due the Qiagen PowerFecal kit eluting in 50uL elution buffer while the ZymoBIOMICS extraction kit recommends eluting in 20uL elution buffer. Qiagen PowerFecal kit again showed very little variability between replicates (n=2) compared to the ZymoBIOMICS extraction kit for the undiluted sample. From a 1:10 dilution both kits produced a similar DNA concentration. From 1:100 dilution onward, both kits had reached a baseline just above the limit of detection for all dilutions ([Figure 4](#tissue_homogenise)).

<a name="tissue_homogenise"></a>
![tissue_homogenise]({{ site.baseurl }}/assets/images/white_001/tissue_lyser_homogenisation.png)
**Figure 4. Nanodrop DNA quantification of DNA extracted from a serial dilution of the ZymoBIOMICS Microbial Community Standard using the Qiagen Tissue Lyser II.**

### Sequencing
Samples in table 2 were submitted for sequencing. Shotgun metagenomic sequencing was performed on a Illumina NovaSeq 6000 at the Oxford Genomics Centre (Q190521/P190535) submitted June 2019.

<table>
  {% for row in site.data.INT001_shotgun %}
    {% if forloop.first %}
    <tr>
      {% for pair in row %}
        <th>{{ pair[0] }}</th>
      {% endfor %}
    </tr>
    {% endif %}
    {% tablerow pair in row %}
      {{ pair[1] }}
    {% endtablerow %}
  {% endfor %}
</table>

### Processing
Dr Matt Jackson kindly performed the analysis. Adapter trimming using trimgalore in the cgatflow readqc pipeline. Pushed through the MetaSequencing pipelines (with no genome filter, just rRNA) and using the smaller Loman lab Kraken2 database.

### Analyses
Analyses can be found in /gfs/devel/majackson/proj038/OCMS_Metagenomic_Standards.

### Shotgun Sequencing Analysis
Analysis of the shotgun sequencing of the undiluted and 1:10 dilution of the ZymoBIOMICS community standard revealed that all microbes within the standard could be recovered in all conditions. In this analysis, sample counts are not normalised to genome size, therefore only a direct comparison to DNA input (i.e. the proportion of each microbe within the standard - DNATrue) is made.

No other microbes and little unassigned taxon indicated no contaminants were introduced during the extraction and sequencing process.

Abundance analysis showed that despite recovering all of the microbes were recovered, of the microbes that were recovered, clear differences in the abundances could be seen when compared to the true input values ([Figure 5](#rel_abund)).

<a name="rel_abund"></a>
![rel_abund]({{ site.baseurl }}/assets/images/white_001/relative_abundance.png)
**Figure 5. Abundances of the different bacterial sequences within each of the samples.**

Principle component analysis allowed comparison of the different variables to determine which condition drove any differences observed.

### Sample Dilution
PCA analysis showed a cluster of 4 conditions which grouped around the DNATrue value and 4 samples which separated from the DNATrue reference value ([Figure 6](#pca_dilution)). This clustering suggested that there was a group of conditions which adequately extracted the DNA from the standard to such an extent that the sequencing analysis clustered closely to the theoretical input of the community standard.

Both of these clusters comprised of a mix of undiluted and 1:10 diluted samples indicating that diluting the neat standard 10x did not affect the sequencing output.

<a name="pca_dilution"></a>
![pca_dilution]({{ site.baseurl }}/assets/images/white_001/pca_dilution.png)
**Figure 6. Principle Component Analysis of all samples for shotgun metagenomic sequencing coloured by dilution.**

### Extraction kit
Both the cluster of 4 samples near the DNATrue reference value and the samples spread further from the DNATrue value comprised of a mixture of samples extracted by the Qiagen PowerFecal Pro kit and the ZymoBIOMICS DNA Microprep kit. The closest samples to the DNATrue reference value were extracted by the Qiagen PowerFecal Pro kit ([Figure 7](#pca_extraction)).

This indicated that the sample extraction kit does not impact on the DNA extracted from the community standard and the sequencing output.

<a name="pca_extraction"></a>
![pca_extraction]({{ site.baseurl }}/assets/images/white_001/pca_extraction.png)
**Figure 7. Principle Component Analysis of all samples for shotgun metagenomic sequencing coloured by extraction kit.**

### Homogeniser
The two different homogenisation protocols; Qiagen Tissue Lyser II versus Qiagen Vortex Adapter showed a significant difference (P<0.001) in microbial sample composition ([Table 3](#linear_model)). The samples extracted by the Qiagen Vortex Adapter all clustered closely to the DNATrue reference value. The samples extracted by the Qiagen Tissue Lyser II clustered away from the DNATrue reference value, although still clustered together on PC1 ([Figure 8 ](#pca_homogenise)).

<a name="pca_homogenise"></a>
![pca_homogenise]({{ site.baseurl }}/assets/images/white_001/pca_homogenisation.png)
**Figure 8. Principle Component Analysis of all samples for shotgun metagenomic sequencing coloured by homogeniser.**

<a name="linear_model"></a>
![linear_model]({{ site.baseurl }}/assets/images/white_001/model_fit.png)
**Table 3. Statistical analysis of the different variables tested**
### Summary
* Qiagen kit produced the most consistent DNA concentration between replicates for both homogeniser protocols although a small n number means significance cannot be determined.
* All microbes within the ZymoBIOMICS Community Standard could be recovered in all conditions tested, although difference in abundances were observed.
* The Qiagen Tissue Lyser II showed a significant difference in the abundances of the microbial community standard sequencing results compared to the DNATrue value. Whereas the samples extracted with the Qiagen Vortex Adapter clustered closely with the DNATrue reference value.
* Extraction kit did not show a significant difference between reference value and sample value. However, samples extracted by the Qiagen PowerFecal kit cluster the closest to the DNATrue reference value when using optimal homogenisation protocol.

[zymo]: http://www.zymoresearch.com/
