
<!DOCTYPE html>
<html lang="en">
<head>
    <title>OCMSExplorer Tutorial - OCMS blog</title>
    <!-- Meta -->
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="This is where we, the OCMS, share our findings, perspectives, and training resources pertaining to microbiome research.">
    <meta name="author" content="ocms@kennedy.ox.ac.uk">
    <link rel="shortcut icon" href="favicon.ico">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'>
    <!-- FontAwesome JS -->
    <script defer src="https://use.fontawesome.com/releases/v5.1.1/js/all.js" integrity="sha384-BtvRZcyfv4r0x/phJt9Y9HhnN5ur1Z+kZbKVgzVBAlQZX4jvAuImlIz+bG7TS00a" crossorigin="anonymous"></script>
    <!-- Global CSS -->
    <link rel="stylesheet" href="assets/plugins/bootstrap/css/bootstrap.min.css">
    <!-- Plugins CSS -->
    <link rel="stylesheet" href="assets/plugins/elegant_font/css/style.css">
    <link rel="stylesheet" href="assets/plugins/prism/prism.css">
    <link rel="stylesheet" href="assets/plugins/lightbox/dist/ekko-lightbox.css">

    <!-- Theme CSS -->
    <link id="theme-style" rel="stylesheet" href="assets/css/styles.css">

</head>

<body class="">
    <div class="page-wrapper">

        <!-- ******header****** -->
        <header id="header" class="header">
            <div class="container">
                <div class="branding">
                    <h1 class="logo"><a href="/">
                            <span class="icon_documents_alt icon" aria-hidden="true"></span>
                            <span class="text-highlight">OCMS</span>
                            <span class="text-bold">Blog</span>
                        </a></h1>
                </div>
                <ol class="breadcrumb">
                    <li class="breadcrumb-item"><a href="/">Home</a></li>
                    <li class="breadcrumb-item"><a href="/bioinformatics">Bioinformatics Resources</a></li>
                    <li class="breadcrumb-item active">OCMSExplorer Tutorial</li>
                </ol>
            </div><!--//container-->
        </header><!--//header-->

        <!--*****doc-wrapper*****-->
        <div class="doc-wrapper">
            <div class="container">
                <!--doc-header-->
                <div id="doc-header" class="doc-header text-center">
                    <h1 class="doc-title">OCMSExplorer Tutorial</h1>
                    <div class="meta"><i class="fa fa-clock"></i> Updated: 11 May 2020</div>
                </div><!--//doc-header-->

                <!--*****section*****-->
                <div class="doc-body row">
                    <!--doc-content-->
                    <div class="doc-content col-12 order-1">
                        <div class="content-inner">
                            <section id="content-section" class="doc-section">
                                <div class="section-block">
                                    <p><h2 id="about-ocmsexplorer">
        
        
          About OCMSExplorer <a href="#about-ocmsexplorer" class="anchor_heading">#</a>
        
        
      </h2>
<p>OCMSExplorer is a visualization tool designed for 16S rRNA gene sequence data. It is a <code class="language-plaintext highlighter-rouge">R Shiny</code> app that is distributed as a <code class="language-plaintext highlighter-rouge">R</code> package. Even though this tool was created as an extension of the 16S rRNA gene analysis pipeline within OCMS, 16S data generated from other pipelines may be used, given that the input file formats are compatible.</p>
    
      <h2 id="installing-ocmsexplorer">
        
        
          Installing OCMSExplorer <a href="#installing-ocmsexplorer" class="anchor_heading">#</a>
        
        
      </h2>
<p>The app is still under development, but it can be installed from the github page. To install OCMSExplorer, use the following in your R console:</p>

<figure class="highlight"><pre><code class="language-r" data-lang="r"><span class="n">devtools</span><span class="o">::</span><span class="n">install_github</span><span class="p">(</span><span class="s2">"path/to/OCMSExplorer@soft_release"</span><span class="p">)</span></code></pre></figure>

<!--more-->
    
      <h2 id="launching-ocmsexplorer">
        
        
          Launching OCMSExplorer <a href="#launching-ocmsexplorer" class="anchor_heading">#</a>
        
        
      </h2>
<p>The app is launched from your R console by:</p>

<figure class="highlight"><pre><code class="language-r" data-lang="r"><span class="n">library</span><span class="p">(</span><span class="n">OCMSExplorer</span><span class="p">)</span><span class="w">
</span><span class="n">run_app</span><span class="p">()</span></code></pre></figure>

<p>This will open the app in your web browser. If pop-ups have been blocked by your browser, you may have to allow them for the app.</p>
    
      <h2 id="using-ocmsexplorer">
        
        
          Using OCMSExplorer <a href="#using-ocmsexplorer" class="anchor_heading">#</a>
        
        
      </h2>
<p>The app is laid out such that all major tasks in the analysis process are shown across the top, where each task is given its own tab. Sub-tasks are shown in the sidebar on the left.</p>
    
      <h3 id="import-uploading-your-data">
        
        
          Import: Uploading your data <a href="#import-uploading-your-data" class="anchor_heading">#</a>
        
        
      </h3>
<p>There is an example dataset available in the Import tab (toggle the <code class="language-plaintext highlighter-rouge">Example data</code> switch to enable/disable). When enabled, you can browser through all the entire app using this dataset.</p>

<p>As part of the OCMS 16S analysis pipeline, the count and taxonomy tables are exported as a <code class="language-plaintext highlighter-rouge">SQLite</code> database file called <code class="language-plaintext highlighter-rouge">csvdb</code>. This file is uploaded as the <code class="language-plaintext highlighter-rouge">Database file</code>. The upload may take a few minutes if the dataset contains several hundred samples. If your raw 16S data was not processed by the OCMS pipeline, there is a helper function in the <code class="language-plaintext highlighter-rouge">OCMSExplorer</code> package that converts csv or tsv files into a <code class="language-plaintext highlighter-rouge">SQLite</code> database file:</p>

<figure class="highlight"><pre><code class="language-r" data-lang="r"><span class="n">OCMSExplorer</span><span class="o">::</span><span class="n">convert2db</span><span class="p">(</span><span class="s2">"/path/to/count.csv"</span><span class="p">,</span><span class="w"> </span><span class="s2">"path/to/taxonomy.csv"</span><span class="p">)</span></code></pre></figure>

<p>The format of the count table should be: features (ASVs or OTUs) in rows, in a column called <code class="language-plaintext highlighter-rouge">featureID</code>, and samples in columns.</p>

<figure class="highlight"><pre><code class="language-markdown" data-lang="markdown">+-----------+---------+---------+-----+
| featureID | sample1 | sample2 | ... |
+-----------+---------+---------+-----+
| ASV1      | ...     | ...     | ... |
+-----------+---------+---------+-----+
| ...       | ...     | ...     | ... |</code></pre></figure>

<p>The format of the taxonomy table should have the columns as shown below. The column <code class="language-plaintext highlighter-rouge">Taxon</code> is a concatenation of all taxonomy levels (<code class="language-plaintext highlighter-rouge">k_kingdom;p_phylum;c_class;o_order;f_family;g_genus;s_species</code>).</p>

<figure class="highlight"><pre><code class="language-markdown" data-lang="markdown">+-----------+----------+--------+--------+-------+-------+--------+-------+---------+-------+
| featureID | sequence |Kingdom | Phylum | Class | Order | Family | Genus | Species | Taxon |
+-----------+----------+--------+--------+-------+-------+--------+-------+---------+-------+
| ASV1      | ATG...   | ...    | ...    | ...   | ...   | ...    | ...   | ...     | ...   |
+-----------+----------+--------+--------+-------+-------+--------+-------+---------+-------+</code></pre></figure>

<p>The metedata associated with your data is imported from a csv or tsv file. The only restriction on the format of the metadata file is that samples are in rows, in a column called <code class="language-plaintext highlighter-rouge">sampleID</code>. Sample identifiers must be unique and must correspond with the sample columns found in your count table. All columns should have a header.</p>

<p>Once the files have been uploaded, click the <code class="language-plaintext highlighter-rouge">Launch Data</code> button to submit. Preview of the uploaded data is available.</p>
    
      <h3 id="qc-dada2-quality-control">
        
        
          QC: Dada2 Quality Control <a href="#qc-dada2-quality-control" class="anchor_heading">#</a>
        
        
      </h3>

<p>Check out the <a href="https://jekyllrb.com/docs/home">Jekyll docs</a> for more info on how to get the most out of Jekyll. File all bugs/feature requests at <a href="https://github.com/jekyll/jekyll">Jekyllâ€™s GitHub repo</a>. If you have questions, you can ask them on <a href="https://talk.jekyllrb.com/">Jekyll Talk</a>.</p></p>
                                </div>
                            </section>
                        </div><!--//content-inner-->
                    </div><!--//doc-content-->
                </div><!--doc-body-->
            </div><!--container-->
        </div><!--//doc-wrapper-->




    </div><!--//page-wrapper-->

    <footer id="footer" class="footer text-center">
    <div class="container">
        <!--/* This template is released under the Creative Commons Attribution 3.0 License. Please keep the attribution link below when using for your own project. Thank you for your support. :) If you'd like to use the template without the attribution, you can buy the commercial license via our website: themes.3rdwavemedia.com */-->
        <small class="copyright">Designed with <i class="fa fa-heart"></i> by <a href="https://themes.3rdwavemedia.com/" target="_blank">Xiaoying Riley</a> for developers</small>
        
    </div><!--//container-->
</footer><!--//footer-->

    <!-- Main Javascript -->
    <script type="text/javascript" src="assets/plugins/jquery-3.3.1.min.js"></script>
    <script type="text/javascript" src="assets/plugins/bootstrap/js/bootstrap.min.js"></script>
    <script type="text/javascript" src="assets/plugins/prism/prism.js"></script>
    <script type="text/javascript" src="assets/plugins/jquery-scrollTo/jquery.scrollTo.min.js"></script>
    <script type="text/javascript" src="assets/plugins/stickyfill/dist/stickyfill.min.js"></script>
    <script type="text/javascript" src="assets/js/main.js"></script>

</body>
</html>
