<!DOCTYPE html>
<html lang="en-US">
  <head>

    
    <meta charset="UTF-8">

<!-- Begin Jekyll SEO tag v2.6.1 -->
<title>OCMSExplorer Tutorial | OCMS blog</title>
<meta name="generator" content="Jekyll v3.8.5" />
<meta property="og:title" content="OCMSExplorer Tutorial" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="About OCMSExplorer OCMSExplorer is a visualization tool designed for 16S rRNA gene sequence data. It is a R Shiny app that is distributed as a R package. Even though this tool was created as an extension of the 16S rRNA gene analysis pipeline within OCMS, 16S data generated from other pipelines may be used, given that the input file formats are compatible. Installing OCMSExplorer The app is still under development, but it can be installed from the github page. To install OCMSExplorer, use the following in your R console: devtools::install_github(&quot;path/to/OCMSExplorer@soft_release&quot;)" />
<meta property="og:description" content="About OCMSExplorer OCMSExplorer is a visualization tool designed for 16S rRNA gene sequence data. It is a R Shiny app that is distributed as a R package. Even though this tool was created as an extension of the 16S rRNA gene analysis pipeline within OCMS, 16S data generated from other pipelines may be used, given that the input file formats are compatible. Installing OCMSExplorer The app is still under development, but it can be installed from the github page. To install OCMSExplorer, use the following in your R console: devtools::install_github(&quot;path/to/OCMSExplorer@soft_release&quot;)" />
<link rel="canonical" href="http://localhost:4000/OCMS-blog/bioinformatics/OCMSExplorer" />
<meta property="og:url" content="http://localhost:4000/OCMS-blog/bioinformatics/OCMSExplorer" />
<meta property="og:site_name" content="OCMS blog" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2020-05-07T00:00:00+01:00" />
<script type="application/ld+json">
{"url":"http://localhost:4000/OCMS-blog/bioinformatics/OCMSExplorer","headline":"OCMSExplorer Tutorial","dateModified":"2020-05-07T00:00:00+01:00","datePublished":"2020-05-07T00:00:00+01:00","mainEntityOfPage":{"@type":"WebPage","@id":"http://localhost:4000/OCMS-blog/bioinformatics/OCMSExplorer"},"description":"About OCMSExplorer OCMSExplorer is a visualization tool designed for 16S rRNA gene sequence data. It is a R Shiny app that is distributed as a R package. Even though this tool was created as an extension of the 16S rRNA gene analysis pipeline within OCMS, 16S data generated from other pipelines may be used, given that the input file formats are compatible. Installing OCMSExplorer The app is still under development, but it can be installed from the github page. To install OCMSExplorer, use the following in your R console: devtools::install_github(&quot;path/to/OCMSExplorer@soft_release&quot;)","@type":"BlogPosting","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->

    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="theme-color" content="#157878">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
    <link rel="stylesheet" href="/OCMS-blog/assets/css/style.css?v=a9c50120f8ae6d5d30d82fe4688a618dc007e7af">
  </head>
  <body>
    <header class="page-header" role="banner">
      <h1 class="project-name">OCMSExplorer Tutorial</h1>
      <br>
      <a href="/" class="btn">Home</a>
      <a href="/bioinformatics" class="btn">Bioinformatics Resources</a>
      <br>

    </header>

    <main id="content" class="main-content" role="main">
    <br>
      <h2 id="about-ocmsexplorer">About OCMSExplorer</h2>
<p>OCMSExplorer is a visualization tool designed for 16S rRNA gene sequence data. It is a <code class="language-plaintext highlighter-rouge">R Shiny</code> app that is distributed as a <code class="language-plaintext highlighter-rouge">R</code> package. Even though this tool was created as an extension of the 16S rRNA gene analysis pipeline within OCMS, 16S data generated from other pipelines may be used, given that the input file formats are compatible.</p>

<h2 id="installing-ocmsexplorer">Installing OCMSExplorer</h2>
<p>The app is still under development, but it can be installed from the github page. To install OCMSExplorer, use the following in your R console:</p>

<figure class="highlight"><pre><code class="language-r" data-lang="r"><span class="n">devtools</span><span class="o">::</span><span class="n">install_github</span><span class="p">(</span><span class="s2">"path/to/OCMSExplorer@soft_release"</span><span class="p">)</span></code></pre></figure>

<!--more-->

<h2 id="launching-ocmsexplorer">Launching OCMSExplorer</h2>
<p>The app is launched from your R console by:</p>

<figure class="highlight"><pre><code class="language-r" data-lang="r"><span class="n">library</span><span class="p">(</span><span class="n">OCMSExplorer</span><span class="p">)</span><span class="w">
</span><span class="n">run_app</span><span class="p">()</span></code></pre></figure>

<p>This will open the app in your web browser. If pop-ups have been blocked by your browser, you may have to allow them for the app.</p>

<h2 id="using-ocmsexplorer">Using OCMSExplorer</h2>
<p>The app is laid out such that all major tasks in the analysis process are shown across the top, where each task is given its own tab. Sub-tasks are shown in the sidebar on the left.</p>

<h3 id="import-uploading-your-data">Import: Uploading your data</h3>
<p>There is an example dataset available in the Import tab (toggle the <code class="language-plaintext highlighter-rouge">Example data</code> switch to enable/disable). When enabled, you can browser through all the entire app using this dataset.</p>

<p>As part of the OCMS 16S analysis pipeline, the count and taxonomy tables are exported as a <code class="language-plaintext highlighter-rouge">SQLite</code> database file called <code class="language-plaintext highlighter-rouge">csvdb</code>. This file is uploaded as the <code class="language-plaintext highlighter-rouge">Database file</code>. The upload may take a few minutes if the dataset contains several hundred samples. If your raw 16S data was not processed by the OCMS pipeline, there is a helper function in the <code class="language-plaintext highlighter-rouge">OCMSExplorer</code> package that converts csv or tsv files into a <code class="language-plaintext highlighter-rouge">SQLite</code> database file:</p>

<figure class="highlight"><pre><code class="language-r" data-lang="r"><span class="n">OCMSExplorer</span><span class="o">::</span><span class="n">convert2db</span><span class="p">(</span><span class="s2">"/path/to/count.csv"</span><span class="p">,</span><span class="w"> </span><span class="s2">"path/to/taxonomy.csv"</span><span class="p">)</span></code></pre></figure>

<p>The format of the count table should be: features (ASVs or OTUs) in rows, in a column called <code class="language-plaintext highlighter-rouge">featureID</code>, and samples in columns.</p>

<figure class="highlight"><pre><code class="language-markdown" data-lang="markdown">+-----------+---------+---------+-----+
| featureID | sample1 | sample2 | ... |
+-----------+---------+---------+-----+
| ASV1      | ...     | ...     | ... |
+-----------+---------+---------+-----+
| ...       | ...     | ...     | ... |</code></pre></figure>

<p>The format of the taxonomy table should have the columns as shown below. The column <code class="language-plaintext highlighter-rouge">Taxon</code> is a concatenation of all taxonomy levels (<code class="language-plaintext highlighter-rouge">k_kingdom;p_phylum;c_class;o_order;f_family;g_genus;s_species</code>).</p>

<figure class="highlight"><pre><code class="language-markdown" data-lang="markdown">+-----------+----------+--------+--------+-------+-------+--------+-------+---------+-------+
| featureID | sequence |Kingdom | Phylum | Class | Order | Family | Genus | Species | Taxon |
+-----------+----------+--------+--------+-------+-------+--------+-------+---------+-------+
| ASV1      | ATG...   | ...    | ...    | ...   | ...   | ...    | ...   | ...     | ...   |
+-----------+----------+--------+--------+-------+-------+--------+-------+---------+-------+</code></pre></figure>

<p>The metedata associated with your data is imported from a csv or tsv file. The only restriction on the format of the metadata file is that samples are in rows, in a column called <code class="language-plaintext highlighter-rouge">sampleID</code>. Sample identifiers must be unique and must correspond with the sample columns found in your count table. All columns should have a header.</p>

<p>Once the files have been uploaded, click the <code class="language-plaintext highlighter-rouge">Launch Data</code> button to submit. Preview of the uploaded data is available.</p>

<h3 id="qc-dada2-quality-control">QC: Dada2 Quality Control</h3>

<p>Check out the <a href="https://jekyllrb.com/docs/home">Jekyll docs</a> for more info on how to get the most out of Jekyll. File all bugs/feature requests at <a href="https://github.com/jekyll/jekyll">Jekyllâ€™s GitHub repo</a>. If you have questions, you can ask them on <a href="https://talk.jekyllrb.com/">Jekyll Talk</a>.</p>



      <footer class="site-footer">
        
          <span class="site-footer-owner"><a href="https://github.com/OxfordCMS/OCMS-blog">OCMS-blog</a> is maintained by <a href="https://github.com/OxfordCMS">OxfordCMS</a>.</span>
        
        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a>.</span>
      </footer>
    </main>
  </body>
</html>
